ui/app/styles/components/form-section.scss

```diff
@@ -7,7 +7,8 @@
   }
 }
 
-.field:first-child .form-section {
+.field:first-child .form-section,
+.box > .field > .field.form-section.string-list {
```

Comment:
I added this to remove `hideFormSection` from model attrs since whenever the string-list displays within a toggle we want to remove form-section styling

Reply:
Nice I like the class approach better now that it's explicitly targeting string-list